odoo/exceptions.py

```diff
@@ -47,7 +47,13 @@ def __init__(self, msg):
 
 
 # deprecated due to collision with builtins, kept for compatibility
-Warning = UserError
+class Warning(UserError):
+    def __init__(self, *args, **kwargs):
+        _logger.warning(
+            "Warning is a deprecated alias to UserError and is going "
+            "to be removed in a future update.",
+            exc_info=DeprecationWarning(), stack_info=True)
```

Comment:
Why not `warnings.warn(..., DeprecationWarning)`.

Reply:
Because warnings are opt-in nobody enable

Reply:
> Because warnings are opt-in nobody enable

https://github.com/odoo/odoo/blob/60b0f6da3420e0bb0607bd73abb4d0a1f5a0a972/odoo/netsvc.py#L131-L132

Reply:
Oh nice !